droidlet/memory/sql_memory.py

```diff
@@ -578,6 +579,16 @@ def get_chat_by_id(self, memid: str) -> "ChatNode":
         """
         return ChatNode(self, memid)
 
+    def get_chat_id(self, speaker_id: str, chat: str) -> str:
```

Comment:
I don't mind this here rn, but I am probably going to torch it in a few commits with the updated search syntax

Reply:
But you are okay with this in this PR ?

Reply:
yes, its fine in this PR